gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/datafetcher/EntityInteractionsFetcherTest.java

```diff
@@ -180,8 +180,7 @@ public void testFetchInteractionsIds_withInteractionFilter() {
     EntityInteractionsFetcher interactionsFetcher =
         new EntityInteractionsFetcher(queryServiceClient, attributeMetadataProvider, queryExecutor);
     List<EntityKey> entityKeys =
-        interactionsFetcher.fetchInteractionsIds(
-            new RequestContext(forTenantId(TENANT_ID)), request);
+        interactionsFetcher.fetchInteractionsIds(entitiesRequestContext, request);
```

Comment:
same issue as above - the mock and call both construct their own request ID with differing request ids, and an eq() matcher is used on line 174. Using the same request context hits the matcher as it used to.